dagql/testdata/introspection.json

```diff
@@ -88,7 +88,7 @@
                 "args": [
                   {
                     "name": "reason",
-                    "value": "use something else"
+                    "value": "\"use something else\""
```

Comment:
Guessing same root cause for quote weirdness?

Reply:
Ah, so this is actually intentional. I updated it to return `.String` instead of `.Raw` - that way it works similarly to `defaultValue`. Otherwise, `string`s just don't have quotes, and can't really be distinguished from `int`s, etc.